CommunityShed/Pages/LoanApplication.aspx.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using CommunityShed.Models;
+using System.Data;
+using System.Data.SqlClient;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.UI;
+using System.Web.UI.WebControls;
+
+namespace CommunityShed.Pages
+{
+    public partial class LoanApplication : System.Web.UI.Page
+    {
+        int lenderId;
+        int borrowerId;
+        int itemId;
+
+
+        private const string GetUserQuery = @"
+           SELECT Id
+           FROM Person
+           WHERE Username = @Username
+       ";
+        protected int? userId;
+        protected void Page_Init(object sender, EventArgs e)
+        {
+            string username = User.Identity.Name;
+            DataTable userTable = DatabaseHelper.Retrieve(GetUserQuery,
+                new SqlParameter("@Username", username));
+            userId = userTable.Rows[0].Field<int>("Id");
+        }
+
+        protected void Page_Load(object sender, EventArgs e)
+        {
+            itemId = int.Parse(Request.QueryString["ID"]);
+            DataTable dt = DatabaseHelper.Retrieve(@"
+                    SELECT i.Name, i.Warnings, p.id AS lenderId
+                    FROM Item i JOIN CommunityPerson cp ON i.CommunityPersonId = cp.Id
+                    JOIN Person p ON cp.PersonId = p.Id
+                    where i.id = @ToolId
+                ", new SqlParameter("@ToolId", itemId));
+            lenderId = dt.Rows[0].Field<int>("lenderId");
```

Comment:
I'd move this line of code into the below `if` statement so you won't get an exception if a row doesn't come back from the database.